index.js

```diff
@@ -1272,6 +1315,8 @@
     ap: ap,
     lift2: lift2,
     lift3: lift3,
+    apFirst: apFirst,
+    apSecond: apSecond,
```

Comment:
One last thing: I think `apFirst` and `apSecond` should come between `ap` and `lift`. Could you move the definitions, the exports, and the tests?